integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/local/transport/test/LocalTransportTestCase.java

```diff
@@ -36,32 +36,9 @@
  */
 public class LocalTransportTestCase extends ESBIntegrationTest {
 
-    private ServerConfigurationManager configurationManagerAxis2;
-    private ServerConfigurationManager configurationManagerCarbon;
-
     @BeforeClass(alwaysRun = true)
     public void deployProxyServices() throws Exception {
         super.init();
-
-        configurationManagerAxis2 =
-                new ServerConfigurationManager(new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN));
-        configurationManagerCarbon =
-                new ServerConfigurationManager(new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN));
-
-        File customAxisConfigAxis2 = new File(getESBResourceLocation() + File.separator +
-                "local" + File.separator + "axis2.xml");
-
-        // enable non-blocking transport axis2 configuration
-        configurationManagerAxis2.applyConfigurationWithoutRestart(customAxisConfigAxis2);
-
-        File customAxisConfigCarbon = new File(getESBResourceLocation() + File.separator +
-                "local" + File.separator + "carbon.xml");
-
-        // restart the server with the changes done to carbon.xml & axis2.xml
-        configurationManagerCarbon.applyConfiguration(customAxisConfigCarbon);
```

Comment:
Isn't the changes in carbon.xml and axis2.xml important here?

Reply:
The changes in carbon.xml are irrelavant, the changes in axis2.xml were moved to a common axis2.xml file